Tests/TestHelpers.swift

```diff
@@ -1,12 +1,12 @@
 import Moya
-import Foundation
 
-extension String {
-    var urlEscaped: String {
-        return self.addingPercentEncoding(withAllowedCharacters: .urlHostAllowed)!
-    }
-}
+#if os(OSX)
+import AppKit
+#else
+import Foundation
```

Comment:
Also didn't know the tests run on Xcode 9 / iOS 11 :) Haven't tried this locally